tests/playwright/e2e/du_tuition_calculator.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { test, expect } from '@du_pw/test';
+import { getRole, logInViaForm, logOutViaUi } from "@du_pw/support/users";
+
+test.describe('DU Tuition Calculator E2E Tests', () => {
+
+  test.beforeEach(async ({ page }) => {
```

Comment:
Makes sense to use `beforeEach` if you have multiple tests going to the same route, but I do not see any tests in this spec.

Need to have at least one test, hopefully testing the calculator, but at least loading the page successfully.